[2024-02-23T02:47:52.854+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dibimbing_example_6.task1 manual__2024-02-23T02:47:50.767972+00:00 [queued]>
[2024-02-23T02:47:52.866+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dibimbing_example_6.task1 manual__2024-02-23T02:47:50.767972+00:00 [queued]>
[2024-02-23T02:47:52.869+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-02-23T02:47:52.881+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): task1> on 2024-02-23 02:47:50.767972+00:00
[2024-02-23T02:47:52.891+0000] {standard_task_runner.py:60} INFO - Started process 96 to run task
[2024-02-23T02:47:52.895+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'dibimbing_example_6', 'task1', 'manual__2024-02-23T02:47:50.767972+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/dibimbing_example_6.py', '--cfg-path', '/tmp/tmp_vp0dp40']
[2024-02-23T02:47:52.901+0000] {standard_task_runner.py:88} INFO - Job 69: Subtask task1
[2024-02-23T02:47:52.966+0000] {task_command.py:423} INFO - Running <TaskInstance: dibimbing_example_6.task1 manual__2024-02-23T02:47:50.767972+00:00 [running]> on host 5c4df0f825b2
[2024-02-23T02:47:53.020+0000] {abstractoperator.py:707} ERROR - Exception rendering Jinja template for task 'task1', field 'op_kwargs'. Template: {'my_variable_value': '{{ var.value.dibimbing_example_6}}', 'execution_date': '{{ data_interval_start }}'}
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 699, in _do_render_template_fields
    rendered_content = self.render_template(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 186, in render_template
    return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 186, in <dictcomp>
    return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 174, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 654, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 131, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 289, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 284, in render_template
    return "".join(nodes)
  File "<template>", line 12, in root
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 105, in __getattr__
    self.var = Variable.get(key, deserialize_json=self._deserialize_json)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable dibimbing_example_6 does not exist'
[2024-02-23T02:47:53.030+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2334, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2465, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2877, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1241, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 699, in _do_render_template_fields
    rendered_content = self.render_template(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 186, in render_template
    return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 186, in <dictcomp>
    return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 174, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 654, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 131, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 289, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 284, in render_template
    return "".join(nodes)
  File "<template>", line 12, in root
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 326, in getattr
    value = getattr(obj, attribute)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 105, in __getattr__
    self.var = Variable.get(key, deserialize_json=self._deserialize_json)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/variable.py", line 143, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable dibimbing_example_6 does not exist'
[2024-02-23T02:47:53.053+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=dibimbing_example_6, task_id=task1, execution_date=20240223T024750, start_date=20240223T024752, end_date=20240223T024753
[2024-02-23T02:47:53.084+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 69 for task task1 ('Variable dibimbing_example_6 does not exist'; 96)
[2024-02-23T02:47:53.124+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-23T02:47:53.177+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
